cmd/zfs/zfs_main.c

```diff
@@ -6859,7 +6859,7 @@ print_holds(boolean_t scripted, int nwidth, int tagwidth, nvlist_t *nvl,
 			if (scripted) {
 				if (parsable) {
 					(void) printf("%s\t%s\t%ld\n", zname,
-					    tagname, time);
+					    tagname, (unsigned long)time);
```

Comment:
You want `(long_t)` here since, the format specifier is signed.  Or switch it to `%lu`.

Reply:
That's what I thought at first.  But then I found other places in the code that cast to `unsigned long`, despite using `%ld`.  So I thought that was the preferred style.  I guess not.  I'll change it back.

Reply:
Sounds like we should fix those too.